testsuite/testsuite.c

```diff
@@ -53,6 +53,16 @@ static const struct {
 					OVERRIDE_LIBDIR "delete_module.so" },
 };
 
+static const char *test_rootfs(const struct test *t)
+{
+	static char dirname[PATH_MAX];
```

Comment:
why is this a static var?

Reply:
The idea is the fall outs of it being static are not (afaict) applicable here and having it dynamically allocated will make `test_export_environ()` even hairier.

Don't have a strong opinion though.